### PR TITLE
Updated version & plugin version update

### PR DESCRIPTION
**Pull request by PurushYM**

### CODE REVIEW

File: ymchat-react-native.podspec
Comment (kaunteya) [inline ymchat-react-native.podspec:19]: Looks good. This ensures that we dont have to update this line until we release 2.0
@PurushYM

**Review by kaunteya — APPROVED** [review]
@PurushYM please wait for sukrits approval on this

File: ymchat-react-native.podspec
Comment (sukrit-ym) [reply ymchat-react-native.podspec:19]: @kaunteya  or 1.5.x right?

**Review by sukrit-ym — APPROVED** [review]

**Pull request merged**
